README.md

```diff
@@ -31,7 +31,7 @@ To do so, just execute:
  
 ## Updating
 
-Run
+@deprecated: Run
```

Comment:
what's depcreated? let's make sure to update docs as needed

Reply:
I've removed self-update command